### PR TITLE
feat: Add support for broadcast push notifications

### PR DESCRIPTION
**Pull request by cbaker6**
- [x] Add missing header information described in Apple Documentation to support broadcast push notifications: https://developer.apple.com/documentation/UserNotifications/sending-broadcast-push-notification-requests-to-apns
- [x] Add documentation for broadcast headers
- [x] Add missing documentation such as `liveactivity`, and `pushtotalk` which is already supported 
- [x] Lint the project to pass build in CI 

Closes: #162

### CODE REVIEW

File: lib/notification/index.js [hunk 1]
```diff
@@ -84,10 +96,6 @@ Notification.prototype.headers = function headers() {
     headers['apns-topic'] = this.topic;
   }
 
-  if (this.collapseId) {
```
Comment (cbaker6) [inline lib/notification/index.js:87]: Just moved this under the rest of the header id's

File: doc/notification.markdown [hunk 2]
```diff
@@ -190,8 +202,4 @@ The type of the notification. The value of this header is `alert` or `background
 
 The value of this header must accurately reflect the contents of your notification's payload. If there is a mismatch, or if the header is missing on required systems, APNs may delay the delivery of the notification or drop it altogether.
 
-#### notification.collapseId
```
Comment (cbaker6) [inline doc/notification.markdown:193]: Just moved this under the rest of the header id's

**Comment by cbaker6** [comment]
@mtrezza this is ready for review

**Review by mtrezza — APPROVED** [review]
Looks good

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [6.3.0](https://github.com/parse-community/node-apn/releases/tag/6.3.0)

**Pull request merged**